apps/web/src/pages/integrations/components/IntegrationInput.tsx

```diff
@@ -13,6 +14,10 @@ export function IntegrationInput({
   register: any;
 }) {
   if (isNeededToHide(credential.key)) {
```

Comment:
I wonder if we can refactor this to have the PasswordInput if statement not here, but rather on the "string" type input 🤔

Reply:
yes lets create a tech debt for this 😄